app/components/UI/Input.js

```diff
@@ -167,44 +190,49 @@ class Input extends React.Component {
           </Label>
         )}
 
-        <SystemInput
-          p={variant === 'thin' ? 2 : 3}
-          {...rest}
-          field={field}
-          type={type}
-          theme={theme}
-          fieldState={fieldState}
-          ref={this.inputRef}
-          value={!value && value !== 0 ? '' : value}
-          required={required}
-          onChange={e => {
-            setValue(e.target.value)
-            if (onChange) {
-              onChange(e)
-            }
-          }}
-          onBlur={e => {
-            setTouched()
-            // Make the state aware that the element is now focused.
-            const newHasFocus = document.activeElement === this.inputRef.current
-            if (hasFocus !== newHasFocus) {
-              this.setState({ hasFocus: newHasFocus })
-            }
-            if (onBlur) {
-              onBlur(e)
-            }
-          }}
-          onFocus={e => {
-            // Make the state aware that the element is no longer focused.
-            const newHasFocus = document.activeElement === this.inputRef.current
-            if (hasFocus !== newHasFocus) {
-              this.setState({ hasFocus: newHasFocus })
-            }
-            if (onFocus) {
-              onFocus(e)
-            }
-          }}
-        />
+        <Flex flexDirection="row" alignItems="center">
```

Comment:
`flexDirection="row"` is a default so can be safely removed

Reply:
updated.